test/liquid/translators/markup_test.exs

```diff
@@ -48,4 +48,19 @@ defmodule Liquid.Translators.MarkupTest do
   test "transforms {:variable_name} tag" do
     assert Markup.literal({:variable_name, "cart"}) == "cart"
   end
+
+  test "transforms {:filters} tag" do
+    assert Markup.literal({:filters, [filter: ["date", {:params, [value: "%w"]}]]}) ==
+             " | date: \"%w\""
+  end
+
+  test "transforms {:assignment} tag" do
+    assert Markup.literal({
+             :params,
+             [
+               assignment: [variable_name: "my_variable", value: "apples"],
+               assignment: [variable_name: "my_other_variable", value: "oranges"]
+             ]
+           }) == ": my_variable: \"apples\", my_other_variable: \"oranges\""
```

Comment:
What is the first semicolon for?

Reply:
`:assignment` is called only on context of `include` tag. `include` has three variants: variables, with, for. `:assignment` is used with `variables` variant and it must create this string:
`'snippet': my_variable: \"apples\", my_other_variable: \"oranges\"`
`with` variant returns:
`'snippet` product with products`
and `for`variant:
`'snippet'` product for products`

The semicolon is mandatory for `variables` variant